.github/workflows/VersionCalPRComment.yml

```diff
@@ -0,0 +1,134 @@
+name: Generate Release Version and Comment PR
+
+on:
+  workflow_dispatch:
+  pull_request_target:
+      types: [opened, labeled, unlabeled, synchronize]
+      paths:
+        - '**.py'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  version-cal:
+    if: contains(github.event.pull_request.labels.*.name, 'auto-cal-version')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set Init Version Message
+        run: |
+          echo "message=$(echo 'Suggested init version: 1.0.0b1 for preview release and 1.0.0 for stable release')" >> $GITHUB_ENV
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+           python-version: "3.10"
+      - name: Checkout CLI extension repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0 # checkout all branches
+          ref: ${{github.event.pull_request.head.ref}}
```

Comment:
There are spaces in `env`, but there are no spaces in `with`. 
Do they need to be consistent?